src/main/java/com/coherentsolutions/training/aqa/java/api/zhavrid/util/HttpClientUtil.java

```diff
@@ -0,0 +1,105 @@
+package com.coherentsolutions.training.aqa.java.api.zhavrid.util;
+
+import org.apache.http.client.methods.*;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+
+import java.util.logging.Logger;
+
+public class HttpClientUtil {
+    private static final Logger logger = Logger.getLogger(ReadWriteRequests.class.getName());
+
+    String url = PropertiesFileReader.getProperty(PropertyKey.URL);
+
+    public void makeGetRequest(String token) {
+
+        try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+            HttpGet httpGet = new HttpGet(url);
+            httpGet.setHeader("Authorization", "Bearer " + token);
+
+            try (CloseableHttpResponse response = httpClient.execute(httpGet)) {
+                int statusCode = response.getStatusLine().getStatusCode();
+                if (statusCode == 200) {
+                    String responseContent = EntityUtils.toString(response.getEntity());
```

Comment:
1) let's use Jackson for deserialization of content to a Java object
2) let's return this entity (use generics to know what exactly should you return)